Apps/WebClient/src/ClientApp/tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "allowSyntheticDefaultImports": true,
     "sourceMap": true,
     "isolatedModules": true,
+    "strictPropertyInitialization": false,
```

Comment:
I think we want this to be true, that way the type checker makes sure all variables are properly initialized

Reply:
Kk